dev/devicelab/lib/tasks/perf_tests.dart

```diff
@@ -769,6 +769,10 @@ class PerfTestWithSkSL extends PerfTest {
       _flutterPath,
       <String>[
         'run',
+        if (deviceOperatingSystem == DeviceOperatingSystem.ios)
+          ...<String>[
+            '--device-timeout', '10',
```

Comment:
Do we also have to apply this to `flutter drive`, or is the timeout there already large enough?

Reply:
Added a timeout to that as well.